terraform/environments/tribunals/cloudfront_nginx.tf

```diff
@@ -0,0 +1,159 @@
+resource "aws_cloudfront_distribution" "tribunals_distribution_nginx" {
+  #checkov:skip=CKV_AWS_86:"Access logging not required for this distribution"
+  #checkov:skip=CKV_AWS_374:"Geo restriction not needed for this public service"
+  #checkov:skip=CKV_AWS_305:"Default root object not required as this is an API distribution"
+  #checkov:skip=CKV_AWS_310:"Single origin is sufficient for this use case"
+  #checkov:skip=CKV2_AWS_47:"Skip Log4j protection as it is handled via WAF"
+  #checkov:skip=CKV2_AWS_46:"Origin Access Identity not applicable as origin is ALB, not S3"
+
+  web_acl_id = aws_wafv2_web_acl.tribunals_web_acl.arn
+
+  logging_config {
+    include_cookies = false
+    bucket          = aws_s3_bucket.cloudfront_logs.bucket_domain_name
+    prefix          = "cloudfront-logs/"
+  }
+
+  aliases = local.is-preproduction ? [
```

Comment:
i now think it would be better switch to

local.is-development  ? [dev option] : siac options